apps/prairielearn/src/models/course-instance-usages.ts

```diff
@@ -0,0 +1,35 @@
+import { loadSqlEquiv, queryAsync } from '@prairielearn/postgres';
+
+const sql = loadSqlEquiv(import.meta.url);
+
+/**
+ * Update the course instance usages for a submission.
+ *
+ * @param param.submission_id The ID of the submission.
+ * @param param.user_id The user ID of the submission.
```

Comment:
Left over from an earlier version?

Reply:
Now used in 5fb1e747660aaca62ac60639d45ab61076eef262